src/test/java/org/utplsql/api/FileMapperIT.java

```diff
@@ -1,15 +1,17 @@
 package org.utplsql.api;
 
+import org.junit.jupiter.api.Test;
 import org.utplsql.api.rules.DatabaseRule;
-import org.junit.Assert;
 import org.junit.Rule;
```

Comment:
Seems like dependency from jUnit; there are no rules in Jupiter

Reply:
It's actually a dependency from junit-jupiter-migrationsupport which is the recommended approach by JUnit 5 as long as you can't completely get rid of Rules - which I can't at that point. I see this as topic for a future refactoring.